packages/docs-reanimated/docs/reanimated-babel-plugin/about.md

```diff
@@ -0,0 +1,217 @@
+---
+id: about
+title: 'About'
+sidebar_label: 'About'
+---
+
+# Reanimated Babel Plugin
+
+This article will explain what exactly is Reanimated Babel Plugin, what are its features and limitations, without going into unnecessary details.
+
+## What is Reanimated Babel Plugin?
+
+Reanimated Babel Plugin is the part of Reanimated that allows you to employ the second JavaScript runtime created by Reanimated, called the UI/Worklet runtime. The plugin transforms your code, injects all the necessary data into it, so that the Worklet runtime can execute the code. This data comes from _worklets_. The process of worklet transformation is called _workletization_.
+
+## What is a worklet?
+
+A worklet is:
+
+- A function that contains a `'worklet'` directive at its very top, i.e.:
+
+```ts
+function foo() {
+  'worklet';
+  console.log('Hello from worklet');
+}
+```
+
+- A function that is _autoworkletizable_, i.e.:
+
+```ts
+useAnimatedStyle(() => {
+  // This function will be ran on the Worklet Runtime,
```

Comment:
```suggestion
  // This function will be ran on the UI thread,
```
i know from the internals point of view a worklet runtime would be ackchyually more correct

but we never use this nomenclature in the documentation. the devs have a mental model of 2 threads - JS and UI.

I feel like saying JS runtime, Worklet runtime is kinda exposing exposing the internals of Reanimated

Reply:
On second thought it makes sense to stay with _UI thread_ here. 👍